plugins/Compressor/CompressorControlDialog.cpp

```diff
@@ -683,7 +681,7 @@ void CompressorControlDialog::drawGraph()
 
 	// Redraw graph
 	m_p.setPen(QPen(m_graphColor, 1));
-	for (int i = 1; i < m_dbRange / COMP_GRID_SPACING + 1; ++i)
+	for (int i = 0; i < m_dbRange / COMP_GRID_SPACING + 1; ++i)
```

Comment:
Is this a off-by-1 bugfix?

Reply:
Not a bug fix, just a personal preference change.  Without this the rightmost line in the grid isn't drawn.